src/storm-cli-utilities/print.cpp

```diff
@@ -0,0 +1,208 @@
+#include "print.h"
+
+#include "storm-version-info/storm-version.h"
+#include "storm/utility/cli.h"
+#include "storm/utility/macros.h"
+
+#include <boost/algorithm/string/replace.hpp>
+#include <ctime>
+
+// Includes for the linked libraries and versions header.
+#include "storm/adapters/IntelTbbAdapter.h"
+
+#ifdef STORM_HAVE_CUDA
+#include <cuda.h>
```

Comment:
These guarded includes here seem to be the same as in the cli.cpp. Is that something that is worth refactoring?

Reply:
The includes in `cli.cpp` were obsolete and I removed them.